### PR TITLE
Adding documentation for Statsig

### PR DESCRIPTION
**Pull request by daniel-statsig**
## Description of the change

> Documentation for the Statsig integration

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

**Comment by ameypv-rudder** [comment]
Thanks for the contribution, @daniel-statsig - while we review the documentation, could you please sign the CLA here: https://rudderlabs.wufoo.com/forms/rudderlabs-contributor-license-agreement. Thanks!

**Comment by daniel-statsig** [comment]
Ok @ameypv-rudder, the CLA has been signed.

**Review by saikumarrs — APPROVED** [review]

File: destinations/analytics/statsig.mdx [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+```
+
+<div class="infoBlock">
+ 
+  For more information on the <code class="inline-code">screen</code> call, refer to the <a href="https://rudderstack.com/docs/rudderstack-api/api-specification/rudderstack-spec/screen/">RudderStack Event Specification</a> guide.
+</div>
+
+## Contact us
+
+**This integration has been built and maintained by Statsig**. For any queries, you can [**contact the Statsig team**](https://statsig.com/contact) or [**get in touch**](mailto:%20docs@rudderstack.com) with us. You can also start a conversation in our [**Slack**](https://rudderstack.com/join-rudderstack-slack-community) community.
```
Comment (ameypv-rudder) [inline destinations/analytics/statsig.mdx:131]: @daniel-statsig please let me know if we can include this contact URL: https://statsig.com/contact or if we should include your community Slack.

Comment (daniel-statsig) [reply destinations/analytics/statsig.mdx:131]: Could you include both? If only one is allowed, the slack link would be preferred. https://statsigcommunity.slack.com/

Comment (ameypv-rudder) [reply destinations/analytics/statsig.mdx:131]: we can add both, no problem!

**Review by ameypv-rudder — APPROVED** [review]

File: destinations/analytics/statsig.mdx [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+  review_body: "Average product, expected much more."
+})
+```
+
+<div class="infoBlock">
+ 
+  For more information on the <code class="inline-code">track</code> call, refer to the <a href="https://rudderstack.com/docs/rudderstack-api/api-specification/rudderstack-spec/track/">RudderStack Event Specification</a> guide.
+</div>
+
+## Screen
```
Comment (saikumarrs) [inline destinations/analytics/statsig.mdx:113]: @ameypv-rudder Statsig has made updates to the transformer to support `group` and `alias` message types as well. Please update the doc accordingly.

Comment (ameypv-rudder) [reply destinations/analytics/statsig.mdx:113]: Sure - thanks for the heads-up here!

Comment (ameypv-rudder) [reply destinations/analytics/statsig.mdx:113]: this is done, @saikumarrs.

**Review by saikumarrs — APPROVED** [review]

**Pull request merged**